data/src/main/java/me/androidbox/data/remote/model/request/EventRequestDto.kt

```diff
@@ -0,0 +1,21 @@
+package me.androidbox.data.remote.model.request
+
+
+/**
+ * /event
+ * POST/Multipart
+ *
+ * Creates a new event. Since Tasky supports offline mode, the IDs for agenda items
+ * need to be generated client-side. Please use UUID.randomUUID().toString() for that.
+ *
+ * Response body: EventDto
+ * */
+data class EventRequestDto(
```

Comment:
I recommend adding SerializedName annotation if using GSON, or  @field:Json if using Moshi

If not, if you end up changing the variable name, or the app gets encrypted (you release it to playstore), then this Dto won't work

This is to all your DTO's. I'll just leave 1 comment as to not disturb the whole pr

Reply:
@MKiperszmidSF 
Is it enough just to annotate all the data classes with the following?

`@JsonClass(generateAdapter = true)`

As the field names are the same as what is expected from the JSON is it good practice to still do this to every field in all the data classes?

Reply:
If the field names are the same, you don't need to add any annotations. I personally only use annotations if the name from the JSON field would be better understandable with a different name